generator/README.md

```diff
@@ -37,12 +37,9 @@ If you would like to run the generator in docker to generate your `snmp.yml` con
 The Docker image expects a directory containing the `generator.yml` and a directory called `mibs` that contains all MIBs you wish to use.
 
 This example will generate the example `snmp.yml` which is included in the top level of the snmp_exporter repo:
+
 ```sh
-make mibs
-docker build -t snmp-generator .
-docker run -ti \
-  -v "${PWD}:/opt/" \
-  snmp-generator generate
+docker-generate
```

Comment:
Shouldn't this be `make docker-generate`?

Reply:
My bad, typo :)
Fixed.